sysutils/node_exporter/Makefile

```diff
@@ -2,7 +2,7 @@
 # $FreeBSD$
 
 PORTNAME=	node_exporter
-PORTVERSION=	0.17.0
+PORTVERSION=	0.18.1
```

Comment:
You shouldn't touch `sysutils/node_exporter` port here. This is a port provided by FreeBSD, if you need to upgrade it, it must be done on FreeBSD ports tree.